cells.py

```diff
@@ -30,9 +30,7 @@
 
 from gettext import gettext as _
 
-###############################################################################
 # Game
```

Comment:
These are supposed to be removed too as the code makes it obvious what it is.

Reply:
Do you mean the comments above of the functions?